test/verify/check-kdump

```diff
@@ -60,6 +60,15 @@ class TestKdump(MachineCase):
         b = self.browser
         m = self.machine
 
+        if self.machine.image == "rhel-8-2-distropkg":
+            self.primary_btn_class = '.btn-primary'
```

Comment:
This happens several times, feel free to move it into testlib.py's MachineCase.

Reply:
Done